src/Admin/Filter/CollectionFilter.php

```diff
@@ -0,0 +1,124 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\ClassificationBundle\Admin\Filter;
+
+use Sonata\AdminBundle\Datagrid\ProxyQueryInterface;
+use Sonata\AdminBundle\Form\Type\Filter\DefaultType;
+use Sonata\ClassificationBundle\Model\CollectionInterface;
+use Sonata\ClassificationBundle\Model\CollectionManagerInterface;
+use Sonata\DoctrineORMAdminBundle\Filter\Filter;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+
+final class CollectionFilter extends Filter
+{
+    /**
+     * @var CollectionManagerInterface
+     */
+    private $collectionManager;
+
+    public function __construct(CollectionManagerInterface $collectionManager)
+    {
+        $this->collectionManager = $collectionManager;
+    }
+
+    /**
+     * @param string            $alias
+     * @param string            $field
+     * @param array|string|null $data
```

Comment:
I see everywhere in these bundle checks like is_array for the $data value.

But according the interface, it's always an array
![image](https://user-images.githubusercontent.com/9052536/78362302-2ac2d580-75ba-11ea-8e81-82401252bd8b.png)